docker/Dockerfile.base

```diff
@@ -0,0 +1,8 @@
+FROM phusion/baseimage:focal-1.0.0
```

Comment:
Instead of adding a new file `Dockerfile.base`, I would like to see all the files combined into a single Dockerfile, with different named images.

For example you could change this line to:
```
FROM phusion/baseimage:focal-1.0.0 as base
```

Then later in the docker file use
```
from base as webapp
```